io_scene_godot/converters/multimesh.py

```diff
@@ -0,0 +1,185 @@
+import math
+import bpy
+import mathutils
+
+from ..structures import (
+    Array, NodeTemplate, InternalResource, fix_matrix)
+from .mesh import ArrayMeshResourceExporter
+
+
+def export_multimesh_node(escn_file, export_settings,
+                          obj, parent_gd_node):
+    """Export blender particle to a MultiMeshInstance"""
+    context = bpy.context
+    dg = context.evaluated_depsgraph_get()
+    ob = context.object.evaluated_get(dg)
+
+    ps = ob.particle_systems.active
+    if (ps.settings.instance_collection and

```

Comment:
In particle system params, If "Render - Render As" are switched to "Collection",there maybe are several objects instanced to one particle,but in Godot one MultiMeshInstance just have one object to instance, so  choose the first object in Blender to display as the only one object in Godot's MultiMeshInstance's resource.

And the instance_object stand for "Render - Render As"  "Object" option,this is most similar with Godot's MultiMeshInstance. 

Don't know if Godot will support multiple object instance to one MultiMeshInstance LOL.

Reply:
Thank you very much for the explanation. Now I have some idea of how it works. It does seem that a inadequacy in Godot to make a particle with multiple meshes. 

I agree it makes sense for now to use the first mesh here. 
Could you add a line of comment  here to explain the reason that we only export `collection[0]`. It would be very helpful. Thanks

Reply:
sure